spec.html

```diff
@@ -4276,16 +4292,16 @@ <h1>GetValue ( _V_ )</h1>
         <emu-alg>
           1. ReturnIfAbrupt(_V_).
           1. If Type(_V_) is not Reference, return _V_.
-          1. Let _base_ be GetBase(_V_).
           1. If IsUnresolvableReference(_V_) is *true*, throw a *ReferenceError* exception.
+          1. Let _base_ be _V_.[[Base]].
           1. If IsPropertyReference(_V_) is *true*, then
             1. If HasPrimitiveBase(_V_) is *true*, then
               1. Assert: In this case, _base_ will never be *undefined* or *null*.
```

Comment:
This assert needs to be changed to include `~empty~`, and probably no longer include *undefined* or *null*, since the latter two are never legal values anymore?